frontend/src/metabase/redux/user.js

```diff
@@ -6,24 +6,30 @@ import { CLOSE_QB_NEWB_MODAL } from "metabase/query_builder/actions";
 
 import { UserApi } from "metabase/services";
 
-export const refreshCurrentUser = createAction("REFRESH_CURRENT_USER", () => {
+export const REFRESH_CURRENT_USER = "metabase/user/REFRESH_CURRENT_USER";
+export const refreshCurrentUser = createAction(REFRESH_CURRENT_USER, () => {
     try {
         return UserApi.current();
     } catch (e) {
         return null;
     }
 });
 
-export const loadCurrentUser = createThunkAction("LOAD_CURRENT_USER", () =>
+export const LOAD_CURRENT_USER = "metabase/user/LOAD_CURRENT_USER";
+export const loadCurrentUser = createThunkAction(LOAD_CURRENT_USER, () =>
     async (dispatch, getState) => {
         if (!getState().currentUser) {
             await dispatch(refreshCurrentUser());
         }
     }
 )
 
+export const CLEAR_CURRENT_USER = "metabase/user/CLEAR_CURRENT_USER"
+export const clearCurrentUser = createAction(CLEAR_CURRENT_USER)
+
 export const currentUser = handleActions({
-    ["REFRESH_CURRENT_USER"]: { next: (state, { payload }) => payload },
-    ["AUTH_LOGOUT"]: { next: (state, { payload }) => null },
+    [CLEAR_CURRENT_USER]: { next: (state, payload) => null },
+    [REFRESH_CURRENT_USER]: { next: (state, { payload }) => payload },
+    [CLEAR_CURRENT_USER]: { next: (state, { payload }) => null },
```

Comment:
Looks like `CLEAR_CURRENT_USER` is in here twice and `REFRESH_CURRENT_USER` is missing. Is that intentional?